server/src/main/resources/store.default.conf

```diff
@@ -1,10 +1,17 @@
 store {
+    # ip
+    ip = "127.0.0.1"
     # Here is the server port that Dst-Server listens on.
     listeningPort = 8082
-    # Whether this is the master
-    isMaster = false
     # The number of shards in this store.
     shardsNum = 8
-    # Change slaver ip by this array
-    slaveAddresses = []
+    # Set up mode.
+    mode = "Stand-alone"
+    # mode = "distributed"
+    # Set node id.
+    nodeId = "hello"
+    # Set heartBeat time interval. (ms)
+    heartBeatInterval = 500
```

Comment:
Is this a reasonable default value for heartBeatInterval?, please check this again

Reply:
I agree with @meijies that we should hardcode the `heartBeatInterval` since this it's not a configurable item.